options/sign.go

```diff
@@ -17,15 +17,17 @@ package options
 import "github.com/spf13/cobra"
 
 type SignOptions struct {
-	KeyOptions  KeyOptions
-	DataType    string
-	OutFilePath string
-	InFilePath  string
+	KeyOptions      KeyOptions
+	DataType        string
+	OutFilePath     string
+	InFilePath      string
+	TimestampServer string
 }
 
 func (so *SignOptions) AddFlags(cmd *cobra.Command) {
 	so.KeyOptions.AddFlags(cmd)
 	cmd.Flags().StringVarP(&so.DataType, "datatype", "t", "https://witness.testifysec.com/policy/v0.1", "The URI reference to the type of data being signed. Defaults to the Witness policy type")
 	cmd.Flags().StringVarP(&so.OutFilePath, "outfile", "o", "", "File to write signed data. Defaults to stdout")
 	cmd.Flags().StringVarP(&so.InFilePath, "infile", "f", "", "Witness policy file to sign")
+	cmd.Flags().StringVar(&so.TimestampServer, "timestamp-server", "", "Timestamp Authority Server to use when signing envelope")
```

Comment:
are we supporting only a signle TSA right now

Reply:
supports multiple now.